### PR TITLE
[PLAY-481] Implementing a Sandpack Example

### PR DESCRIPTION
**Pull request by carloslimasd**
#### Breaking Changes

No

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-481)

#### How to test this

In the URL type in /kits/[kit-name]/new
Example: /kits/avatar/new

#### Checklist:

- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [ ] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

File: playbook-website/app/javascript/components/PbKitReact/index.tsx [hunk 1]
```diff
@@ -1,15 +1,42 @@
 import React from 'react'
-
 import { Title } from 'playbook-ui'
+import { Sandpack } from '@codesandbox/sandpack-react'
+import TableSmCode from 'playbook-ui/app/pb_kits/playbook/pb_table/docs/_table_sm'
```
Comment (jasoncypret) [inline playbook-website/app/javascript/components/PbKitReact/index.tsx:4]: We can't do this as this is not scaleable

File: playbook/app/pb_kits/playbook/pb_table/docs/_table_sm.jsx [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+- We could use raw-loader from Asset Modules: https://webpack.js.org/guides/asset-modules/
+- Or we could just export the string in the file as I did below.
 
-import Table from '../_table'
+But even so, in the end, we would have to change the document a little bit to turn it into "real code" without errors.
 
-const TableSm = (props) => {
+For example: for this one, I had to remove the parameter "props" (because we were not using it) and change the way that we import the component Table from Playbook.
+*/
+const TableSmCode = `import React from 'react'
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_table/docs/_table_sm.jsx:12]: We can't require our examples to be strings to display. Developers wouldn't be able to reuse the code in their environments

**Review by jasoncypret — CHANGES_REQUESTED** [review]

File: playbook-website/app/javascript/site_styles/_site-style.scss [hunk 3]
```diff
@@ -16,6 +16,10 @@ body {
   }
 }
 
+iframe body {
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_site-style.scss:19]: Can we make this into a selector?

File: playbook/app/pb_kits/playbook/pb_docs/kit_example.html.erb [hunk 4]
```diff
@@ -1,28 +1,32 @@
...
-    <%= pb_rails("caption", props: { text: example_title }) %>
-    <br />
-    <%= example %>
-    <br>
-  </div>
-
-  <% if show_code %>
-    <div class="markdown pb--kit-example-markdown <%= dark ? "dark" : "" %>">
-      <%= markdown(description) %>
+<% if type == "react"%>
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_docs/kit_example.html.erb:1]: Can we leave the existing code in tact?

File: playbook-website/app/javascript/site_styles/_site-style.scss (hunk 3 above)
Comment (carloslimasd) [reply playbook-website/app/javascript/site_styles/_site-style.scss]: This code was a test. It has been removed.

**Review by jasoncypret — APPROVED** [review]

**Comment by jasoncypret** [comment]
@jasperfurniss Can't this just ship. No release needed

**Pull request merged**